src/app/node-data/basic/provider/kubevirt/component.ts

```diff
@@ -60,14 +60,14 @@ export class KubeVirtBasicNodeDataComponent extends BaseFormValidator implements
       [Controls.CPUs]: this._builder.control('1', Validators.required),
       [Controls.Memory]: this._builder.control('2Gi', [
         Validators.required,
-        Validators.pattern(this._sizeSuffixPattern),
+        Validators.pattern(this._memorySizePattern),
       ]),
-      [Controls.Namespace]: this._builder.control('', Validators.required),
+      [Controls.Namespace]: this._builder.control('kube-system', Validators.required),
```

Comment:
Have you checked the MD edit dialog after creating the cluster? This dialog is also displayed there and I think that if you hardcode `kube-system` here then it will always use it in the edit instead of taking the actually specified namespace.

Reply:
It works as expected, attached a GIF in PR description PTA.

Reply:
Great. LGTM then